kong/api/routes/certificates.lua

```diff
@@ -12,6 +12,37 @@ return {
         self.params.snis = nil
       end
 
+      -- dont add the certificate or any snis if we have an SNI conflict
+      -- its fairly inefficient that we have to loop twice over the datastore
+      -- but no support for OR queries means we gotsta!
+      local seen = {}
+      if type(snis) == "table" then
+        for i = 1, #snis do
+          local sni_name = snis[i]
```

Comment:
`ipairs` is a bit slower than an iterative loop, due to the function call overhead, but given the likely scope of this, it probably wouldn't make much difference either way :) let me know if this is a blocker (doesnt seem like it)